octoprint_psucontrol/__init__.py

```diff
@@ -64,7 +64,7 @@ def __init__(self):
         self._idleTimer = None
         self._waitForHeaters = False
         self._skipIdleTimer = False
-        self._configuredGPIOPins = {}
+        self._configuredGPIOPins: Dict[str, periphery.CdevGPIO] = {}

```

Comment:
This plugin still supports older Python 2.7 installations which doesn't support type annotations.

Reply:
Sorry about that, I did not realize that octoprint still has support for python 2.7.